beacon/deploy/conf.py

```diff
@@ -0,0 +1,125 @@
+"""Beacon Configuration."""
+
+#
+# Beacon general info
+#
+beacon_id = 'org.nbis.ga4gh-approval-beacon-test' # ID of the Beacon
+beacon_name = 'European Genomic Data Infrastructure Swedish Test Beacon' # Name of the Beacon service
+api_version = 'v2.0.0'  # Version of the Beacon implementation
+uri = 'https://beacon.gdi.nbis.se/api/'
+
+#
+# Beacon granularity
+#
+default_beacon_granularity = "record"
+max_beacon_granularity = "record"
+
+#
+#  Organization info
+#
+org_id = 'NBIS'  # Id of the organization
+org_name = 'National Bioinformatics Infrastructure Sweden (NBIS)'  # Full name
+org_description = ('NBIS is a distributed national bioinformatics infrastructure, '
+                   'supporting life sciences in Sweden.')
+org_adress = ('Uppsala Biomedicine Center (BMC) '
+              'Husargatan 3, '
+              '751 23 Uppsala, Sweden')
+org_welcome_url = 'https://gdi.nbis.se/'
+org_contact_url = 'mailto:info@gdi.nbis.se'
+org_logo_url = 'https://gdi.onemilliongenomes.eu/images/gdi-logo.svg'
+org_info = ''
+
+#
+# Project info
+#
+description = r"This Beacon is based on two synthetic datasets. The first is the dataset EGAD00001003338 at EGA archive and it contains 2504 samples including genetic data based on 1K Genomes data, and 76 individual attributes and phenotypic data derived from UKBiobank. The second one, is the Heilsa synthetic data, including 14 individuals."
```

Comment:
typo before quotes?

Reply:
The `r` indicates a "raw string" (in which backslashes won't be interpreted).